src/java.net.http/share/classes/module-info.java

```diff
@@ -25,7 +25,125 @@
+ * then the Java Logging API logs all possible HTTP Client events:<br>
+ * "errors,requests,headers,frames:control:data:window,ssl,trace,channel"<br>
+ * Note that you can replace control:data:window with all. The name of the logger is
+ * "jdk.httpclient.HttpClient", and all logging is at level INFO.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.keepalive.timeout}</b> (default: 30)<br>
+ * The number of seconds to keep idle HTTP connections alive in the keep alive cache. This property
+ * applies to both HTTP/1.1 and HTTP/2. The value for HTTP/2 can be overridden with the
+ * jdk.httpclient.keepalive.timeout.h2 property.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.keepalive.timeout.h2}</b> (default: see below)<br>
+ * The number of seconds to keep idle HTTP/2 connections alive. If not set, then the
+ * jdk.httpclient.keepalive.timeout setting is used.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.maxframesize}</b> (default: 16384 or 16kB)<br>
+ * The HTTP/2 client maximum frame size in bytes. The server is not permitted to send a frame larger than this.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.maxstreams}</b> (default: 100)<br>
+ * The maximum number of HTTP/2 push streams that the client will permit servers to open simultaneously.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.receiveBufferSize}</b> (default: operating system default)<br>
+ * The HTTP client <a href="../java.base/java/net/StandardSocketOptions.html#SO_RCVBUF">socket receive
+ * buffer size</a> in bytes.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.redirects.retrylimit}</b> (default: 5)<br>
+ * The maximum number of attempts to send a HTTP request when redirected or any failure occurs for any reason.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.websocket.writeBufferSize}</b> (default: 16384 or 16kB)<br>
+ * The buffer size used by the web socket implementation for socket writes.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.windowsize}</b> (default: 16777216 or 16 MB)<br>
+ * The HTTP/2 client stream window size in bytes.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.auth.retrylimit}</b> (default: 3)<br>
+ * The number of attempts the Basic authentication filter will attempt to retry a failed authentication.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.sendBufferSize}</b> (default: operating system default)<br>
+ * The HTTP client socket <a href="../java.base/java/net/StandardSocketOptions.html#SO_SNDBUF">send
+ * buffer size</a>. Values less than or equal to zero are ignored.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.internal.httpclient.disableHostnameVerification}</b> (default: false)<br>
+ * If true, hostname verification in SSL certificates is disabled. This is a system property only and not
+ * available in {@code conf/net.properties}. It is provided for testing purposes only.
```

Comment:
From what I can see in the code `jdk.internal.net.http.common.Utils#hostnameVerificationDisabledValue()`, we consider an empty value for this property to imply `true`. Perhaps we should note that here.

Reply:
Yes, agreed!